yoti-sdk/src/main/java/com/yoti/mobile/android/sdk/YotiSDKButton.java

```diff
@@ -135,6 +168,10 @@ public void onClick(View v) {
         void onYotiAppNotInstalledError(YotiSDKNoYotiAppException cause);
     }
 
+    public interface OnAppNotInstalledListener {
```

Comment:
I think we should call this interface as `YotiAppStateListener` 🤔

Reply:
I think `AppStateListener` is too generic:)